src/commands/Utilities/afk.js

```diff
@@ -29,22 +29,22 @@ module.exports = {
             const n = flags[1];
             const { length } = displayName;
             switch (type) {
-            case 1: // Set
-                if (length <= 26 && !n) {
-                    message.member.setNickname('[AFK] ' + displayName)
-                        .catch(e => {
-                            if (!(e instanceof DiscordAPIError)) throw e;
-                        });
-                }
-                break;
-            case 2: // Remove before
-                if (length <= 26 && !afk.get(id).flags[1]) {
-                    message.member.setNickname(displayName.slice(6))
-                        .catch(e => {
-                            if (!(e instanceof DiscordAPIError)) throw e;
-                        });
-                }
-                break;
+                case 1: // Set
+                    if (length <= 26 && !n) {
+                        message.member.setNickname('[AFK] ' + displayName)
+                            .catch(e => {
+                                if (!(e instanceof DiscordAPIError)) throw e;
+                            });
+                    }
+                    break;
+                case 2: // Remove before
+                    if (length <= 26 && !afk.get(id).flags[1] && displayName.startsWith('[AFK] ')) {
```

Comment:
```suggestion
                    if (displayName.startsWith('[AFK] ') && !afk.get(id).flags[1]) {
```

Also here, the `displayName.startsWith('[AFK] ')` implies it was already set, therefore `length <= 26` isn't needed (and actually, this might fuck up because it doesn't take the original displayName.
Please remove the first check (not on the case 1 though)